MAVProxy/modules/mavproxy_chat/chat_openai.py

```diff
@@ -99,6 +100,23 @@ def set_api_key(self, api_key_str):
         self.assistant = None
         self.assistant_thread = None
 
+    # cancel the active run
+    def cancel_run(self):
+        # check the active thread and run
+        if (self.assistant_thread and self.latest_run is not None):
+            if (self.latest_run.status != "completed" and "cancelled"):
+
+                # cancel the active run
+                self.client.beta.threads.runs.cancel(
+                    thread_id=self.assistant_thread.id,
+                    run_id=self.run.id
+                )
+            else:
+                if (self.lastest_run.status == "completed"):
```

Comment:
little typo here, "lastest_run" should be "latest_run"

Reply:
Oops!